tests/runFromSimTest.m

```diff
@@ -11,113 +11,22 @@
             % Set WEC-Sim, test and simulink directories
             obj.testDir = fileparts(mfilename('fullpath'));
             obj.wsDir = fullfile(obj.testDir,'..');
-            obj.runFromSimDir = fullfile(obj.testDir,  ...
-                                         'runFromSimulinkTests');
+            obj.runFromSimDir = fullfile(obj.testDir, '..', ...
+                'examples', 'RM3');
         end        
```

Comment:
Are you removing the `example\RM3FromSimulink` directory, and instead running from Simulink with the `example/RM3`? That makes sense to me, but the `example/RM3` case isn't running for me directly from Simulink.

Reply:
This should be fixed, there was a reference to the non-hydro body in the Rigid Body mask that should have been removed

Reply:
Thanks, confirmed that this issue is resolved!